exchange/exchange_test.go

```diff
@@ -1019,15 +1022,25 @@ func newExchangeForTests(t *testing.T, filename string, expectations map[string]
 		}
 	}
 
+	var s struct{}
+	eeac := make(map[string]struct{})
+	for _, c := range []string{"ALA", "AUT", "BEL", "BGR", "HRV", "CYP", "CZE", "DNK", "EST",
```

Comment:
Do we need such a long list for the tests? Can we simplify this?

Reply:
I just copied what we had for the default. I am torn between having a simple setup, and having a setup that more resembles what would be seen in production.

Reply:
I have a stronger preference to keep things simple for the test. Makes it a bit easier to read and functionality the test is just as valid if there are 1 or 2 entries as 20+.